test/_imports/project.yaml

```diff
@@ -0,0 +1,90 @@
+functionEvents:
+  test-function-event-2:
+    Meta:
+      labels:
+        nuclio.io/function-name: test-function-2
+      name: test-function-event-2
+    Spec:
+      attributes:
+        headers:
+          Content-Type: text/plain
+        method: POST
+        path: ""
+      body: test-function-event-2-body
+      displayName: test-function-event-2
+      triggerKind: http
+  test-function-event-1:
+    Meta:
```

Comment:
Please use `meta` (small letters) here and everywhere else (Meta/Spec) it occurs